TT Lab/Editors/ChunkEditor.xaml.cs

```diff
@@ -89,5 +143,169 @@ public List<AssetViewModel> ChunkTree
         {
             get => chunkTree;
         }
+
+        private void MouseMove(Object? sender, MouseEventArgs e)
+        {
+            if (e.LeftButton != MouseButtonState.Pressed)
+            {
+                return;
+            }
+            var pos = e.GetPosition(SceneRenderer.Glcontrol);
+            editingContext.UpdateTransform((float)pos.X, (float)pos.Y);
+        }
+
+        private void MouseDown(Object? sender, MouseEventArgs e)
+        {
+            if (e.LeftButton != MouseButtonState.Pressed)
+            {
+                return;
+            }
+            var pos = e.GetPosition(SceneRenderer.Glcontrol);
+            if (editingContext.transformMode == TransformMode.SELECTION || editingContext.transformAxis == TransformAxis.NONE)
+            {
+                MouseSelect((float)pos.X, (float)pos.Y);
+            }
+            else
+            {
+                editingContext.StartTransform((float)pos.X, (float)pos.Y);
+            }
+        }
+
+        private void MouseUp(Object? sender, MouseEventArgs e)
+        {
+            var pos = e.GetPosition(SceneRenderer.Glcontrol);
+            editingContext.EndTransform((float)pos.X, (float)pos.Y);
+        }
+
+        private void KeyDown(Object? sender, KeyEventArgs e)
+        {
+            var key = e.Key;
+            if (pressedKeys.Contains(key))
+            {
+                return;
+            }
+
+            pressedKeys.Add(key);
+            if (key == Key.LeftAlt) { leftAlt = true; return; }
```

Comment:
We need to track both keys, because if i press both Alt and release one, with simple "Alt = false" program will think that Alt is not pressed.
So we need either Alt counter, or 2 bools to track both right and left keys